packages/relay/src/lib/errors/JsonRpcError.ts

```diff
@@ -238,6 +238,10 @@ export const predefined = {
     code: -39013,
     message: 'Invalid block range',
   }),
+  RECEIVER_SIGNATURE_ENABLED: new JsonRpcError({
+    code: -32000,
+    message: "Receiver's signature is enabled.",
+  }),
```

Comment:
```suggestion
  RECEIVER_SIGNATURE_REQUIRED: new JsonRpcError({
    code: -32000,
    message: "Receiver's signature is required.",
  }),
```

Reply:
Suggestion from the team `Operation not supported when receiver's signature is enabled.`